<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>El_GUI_BANDITO</class>
 <widget class="QMainWindow" name="El_GUI_BANDITO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>el_gui_bandito v.0.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Main_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <widget class="QFrame" name="left_frame">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>210</width>
       <height>700</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>700</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QGroupBox" name="null_gBox">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>el_gui_comrado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="null_toolB_4">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="null_toolB_3">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>null_button</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setting_toolB">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_status_2">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QGroupBox" name="settings_gBox">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="null_toolB_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>null_button</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="null_toolB_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>null_button</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="settings_toolB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="network_stat_line">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>35</height>
            </size>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="right_frame">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>10</y>
       <width>1040</width>
       <height>700</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QToolButton" name="toolButton_8">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>303</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>60</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_7">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>222</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_6">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>141</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>627</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>546</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>465</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>384</y>
        <width>21</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
